### DISCUSSION TITLE
[Feature] Sort People by Name (alphabetically)

### DISCUSSION THREAD

**Discussion by schnillerman** [immich-app/immich#10584] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add sorting by name (alphabetically) to the people view (immich-instalncel.tld/people)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5927

You've just opened four duplicate feature requests.
> I have searched the existing feature requests to make sure this is not a duplicate request.
> [ ] Yes

This checkbox is there for very good reason. Please actually use the search before opening a feature request in the future.

**Reply by schnillerman** [reply]
None of the searchwords I used for those features (_sidecar_ is the offcial term from immich documentation instead of XMP; _sorting_ is a well-known concept of data display, etc.) yielded any results, I'm sorry. But I DID search for existing feature requests. As an example, the fact that issue #8377 doesn't contain _sorting_ as a keyword (where sorting is really the main purpose) isn't my fault.

But I do understand that this caused unnecessary work for you, so I apologize.
